lib/route.coffee

```diff
@@ -207,8 +207,15 @@ class @Route
     # Send response
     endpointContext.response.writeHead statusCode, headers
     endpointContext.response.write body
-    endpointContext.response.end()
-
+    switch statusCode
+        when 401, 403
+            # This is a simple, but not really adequate way to slow down unauthorized scans of the server.
+            # It is not sufficient because hackers can DDOS across multiple threads and systems.
+            timeoutInMilliseconds = 500
+            timeoutInMilliseconds *= (1 + Math.random())
```

Comment:
I think we should change this to
```coffeescript
randomMultiplierUpTo2X = 1 + Math.random()
delayInMilliseconds = 500 * randomMultiplierUpTo2X
```
I'm not in love with my variable name, but it gets rid of the magic number, and makes life a little easier for the math-challenged devs :) Also, I prefer "delay" over "timeout" for the other variable name.